src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -90,6 +90,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('no_wait', help="Do not wait for the run to complete and return immediately after queuing the run.", action='store_true')
         c.argument('no_format', help="Indicates whether the logs should be displayed in raw format", action='store_true')
         c.argument('os_type', options_list=['--os'], help='The operating system type required for the build.', arg_type=get_enum_type(OsType))
+        c.argument('platform', options_list=['--platform'], help='The platform required for this build.')
```

Comment:
Options_list not needed here since the CLI will automatically infer this.

Reply:
doone